<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>242</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="menuListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/general_settings.png</normaloff>:/icons/general_settings.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Compression</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/compression_settings.png</normaloff>:/icons/compression_settings.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Language</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/locale_settings.png</normaloff>:/icons/locale_settings.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="optionStackedWidget">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="generalPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="title">
          <string>Default Path</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="margin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="useDefaultPathCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Use a default output directory if none is specified</string>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="defaultOutputLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="browseToolButton">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="title">
          <string>Default Image View</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="originalSizeRadio">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Original Size</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string>buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fitWindowRadio">
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Fit Window</string>
              </property>
              <attribute name="buttonGroup">
               <string>buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="scanSubDirCheckBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Scan subdirectories when you open a folder</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="loadLastProfileCheckBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Load the last used profile at startup</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="promptBeforeExitCheckBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Prompt before exit</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkUpdatesCheckBox">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Check for uptades at startup</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="compressionPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="keepExifCheckBox">
         <property name="text">
          <string>Keep EXIF Information while compressing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keepDateCheckBox">
         <property name="text">
          <string>Keep the original Date Information</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="doNotEnlargeCheckBox">
         <property name="text">
          <string>Skip if the output size is greater than the original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="deleteCheckBox">
         <property name="text">
          <string>Delete the original file from disk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="languagePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="languageTreeWidget">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <column>
            <property name="text">
             <string>Language</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>208</red>
           <green>221</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>208</red>
           <green>221</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>208</red>
           <green>221</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>208</red>
           <green>221</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useDefaultPathCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaultOutputLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>optionStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuListWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>titleLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
